packages/eslint-plugin/src/configs/base.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
      * @see https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin
      */
     ...getNamingConventionRule(),
+    '@fluentui/max-len': 'off',
```

Comment:
this rule doesn't make much, especially with severity "error", 

it was also triggered for new generated component by `create-component`

in general we have prettier to enforce line length so turning this off for new codebase